docker/archive/src.go

```diff
@@ -32,7 +35,7 @@ func (s *archiveImageSource) Reference() types.ImageReference {
 
 // Close removes resources associated with an initialized ImageSource, if any.
 func (s *archiveImageSource) Close() error {
-	return nil
+	return s.Source.Close()
```

Comment:
The `Close` method can be completely removed, Go will automatically use the method in the anonymous `Source` member.

Reply:
fixed